LICENSE.md

```diff
@@ -70,6 +70,7 @@ The following components of Julia's standard library have separate licenses:
 Julia builds the following libraries by default, but does not use them itself:
 
 - [RMATH](http://www.r-project.org/Licenses/)
+- [LDC](https://github.com/ldc-developers/ldc/blob/master/LICENSE)
```

Comment:
good catch. when this was first written the file wasn't sorted, so rebase hadn't noticed that everything else moved